acceptance/setup/packages/pre-suite/010_Install.rb

```diff
@@ -47,3 +47,7 @@
 
 install_packages_on(master, MASTER_PACKAGES)
 install_packages_on(agents, AGENT_PACKAGES)
+
+configure_gem_mirror(master)
+configure_gem_mirror(agents)
+
```

Comment:
is there any reason to separate the calls here, rather than a single
configure_gem_mirror(hosts)?